src/components/home/components/Mids.tsx

```diff
@@ -22,25 +23,31 @@ const Mids = () => {
                         <Row className="mt-3">
                             <Col className="home_introText">
-                                Minimum Information about a Digital Specimen (MIDS) is a specification defining the information elements expected to be present when publishing digitised specimen information at various levels of digitisation. Digital Specimens are online digital representations of their physical counterparts in natural science collections. The definition of digitisation used here is the process of making physical objects digitally available. Levels of digitisation represent a simple categorisation of the type and depth of digitisation achieved by different approaches.
+                                Minimum Information about a Digital Specimen (MIDS) is a specification defining the information elements 
+                                expected to be present when publishing digitised specimen information at various levels of digitisation. 
+                                Digital Specimens are online digital representations of their physical counterparts in natural science collections. 
+                                The definition of digitisation used here is the process of making physical objects digitally available. Levels of 
+                                digitisation represent a simple categorisation of the type and depth of digitisation achieved by different approaches.
                             </Col>
                         </Row>
                     </Col>
                 </Row>
                 <Row className="mt-5">
                     <Col md={{ span: 12 }}>
                         <Row>
-                            {midsLevels.map((level, i) => {
+                            {midsLevels.map((level) => {
                                 return (
-                                    <Col md={{ span: 3 }} key={i}>
+                                    <Col md={{ span: 3 }} key={level}>
                                         <Row>
                                             <Col md={{ span: 10 }}>
                                                 <div>
                                                     <div className="home_midsTitleBlock py-2 bg-primary-dark fw-bold text-white text-center">
                                                         MIDS level {level}
                                                     </div>
                                                     <div className="px-4 py-3 br-bl br-br border-2-primary-dark">
-                                                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                                                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore 
```

Comment:
Maybe add a task on the backlog to fix the text here. See if we can get Elspeth or someone to provide some text on MIDS